src/api/blogService.js

```diff
@@ -1,20 +1,26 @@
-import axios from '@/api/axiosConfig'
+import axios from '@/api/axiosConfig';
 
-export async function getBlogs({ page, perPage, sort }) {
-    let params = {
-        page: page,
-        perPage: perPage,
-        sort: sort
-    };
-    try {
-        let { data } = await axios.get('/api/restify/blogs', { params });
-        return data;
-    } catch (error) {
-        return error;
-    }
+export async function getBlogs(searchTerm, { page, perPage, sort }) {
+  let params = {
+    page: page,
+    perPage: perPage,
+    sort: sort,
+  };
+  try {
+    let { data } = await axios.get(
+      '/api/restify/blogs?sort=-views&search=' + searchTerm,
```

Comment:
you have the sort query in the params object and also search can go there also

Reply:
And the seach endpoint is different {{api}}/api/blogs/advance-search?search=Prof